config/abilities/provider_admin.rb

```diff
@@ -79,10 +79,9 @@
     service.account_id == user.account_id && can?(:manage, :multiple_services) and not service.default_or_last?
   end
 
-  if account.provider_can_use?(:api_as_product)
-    can :manage, BackendApi
-    can :manage, BackendApiConfig
-  end
+  can %i[index show edit update create destroy], BackendApi
```

Comment:
It think that it should be like this:
```suggestion
  can %i[index show edit update new create destroy], BackendApi
```

Because `Provider::Admin::BackendApisController` has the `new` action.

Reply:
My bad, I see that see handle that scenario in `config/initializers/cancan_hacks.rb` checking `show`.